src/main/resources/liquibase/harness/change/changelogs/postgresql/changeCompositeTypeSchema.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="itTest" id="1">
+        <pro:createCompositeType name="myTypeForSchemaTest">
+            <pro:typeAttribute name="attr1" type="integer"/>
+            <pro:typeAttribute name="attr2" type="text" collation="en_US"/>
+        </pro:createCompositeType>
+        <rollback>
```

Comment:
similar to json - remove rollbacks from createCompositeType, changeCompositeTypeSchema, remove 4th changeset